### PR TITLE
Several improvements to Repo Homepage

### PR DESCRIPTION
**Pull request by mmattar**
- Updates features list
- Bolded documentation home for emphasis - now links to documentation readme not directory
- Added blog post URL
- Removed reference to windows installation guide

### CODE REVIEW

**Comment by xiaomaogy** [comment]
Is the 0.3 version also called Beta version as it shows Unity ML-Agents (Beta) as the title？
Also this example environment did we use for neuroevolution?

**Comment by awjuliani** [comment]
@xiaomaogy Yep, still beta. Beta until we reach v1.0. Also, no example for neuroevolution. Just want to call out the fact that this toolkit doesn't preclude the use of evolutionary algorithms over RL.

**Comment by awjuliani** [comment]
@mmattar Should we perhaps link here to a page on community contributions? Something like "if you'd like to get involved..."

File: README.md [hunk 1]
```diff
@@ -18,36 +18,35 @@ in AI can be evaluated on Unity’s rich environments and then made accessible
...
 
 ## Features
-* Unity Engine flexibility and simplicity
-* Flexible single-agent and multi-agent support
-* Multiple visual observations (cameras)
-* Discrete and continuous action spaces
+* Unity environment control from Python
+* 10+ sample Unity environments
+* Support for multiple environment configurations and training scenarios
+* Train memory-enhanced Agents using deep reinforcement learning
```
Comment (awjuliani) [inline README.md]: Maybe call out ODD?

Comment (mmattar) [reply README.md]: We do, see below "Flexible Agent control with On Demand Decision Making" - good enough or should we clarify?

**Comment by mmattar** [comment]
@awjuliani Regarding contributions, we do mention that, but it was part of an earlier PR. Check out the full file and there's a section on Contributing. I'm writing up a contribution guidelines page today.

**Comment by awjuliani** [comment]
@mmattar Great!

**Pull request merged**